code/modules/ruins/lavalandruin_code/techcult.dm

```diff
@@ -325,7 +325,7 @@
 	name = "Adeptus Mechanicus"
 	id = "techcult"
 	description = "An old base, filled with religious fanatics praising the entity they call 'Machine God'."
-	suffix = "icemoon_underground_techcult.dmm"
+	suffix = "icemoon_underground_techcult2.dmm"
```

Comment:
You could've just left a normal name for .dmm, no need to leave "2" in the name when you are replacing the old one.